s3/error/s3errors.go

```diff
@@ -694,23 +700,23 @@ var ErrorCodeResponse = map[S3ErrorCode]S3ErrorStruct{
 		HttpStatusCode: http.StatusInternalServerError,
 	},
 	ErrGetBackendFailed: {
-		AwsErrorCode: "GetBackendFailed",
-		Description: "Backend is not exist, or get it failed.",
+		AwsErrorCode:   "GetBackendFailed",
+		Description:    "Backend is not exist, or get it failed.",
```

Comment:
can we have only codes on backend...as in, for example, AwsErrorCode...this way, the UI can use this error code and show up a language specific description from a file. Having Description coming from backend is generally not a good idea imho, please check

Reply:
The S3 service only return error_code to API service, API service will get description by this error_code, currently we only support English description, so API service just get the description from this error map. But in  the future if we need to do some language specific things, then we can add a configure file, from which API service can get language specific description by error_code. And currently, we leaved English description here, so we can print it in log, which is better for tracing. How do you think about this?

Reply:
Issue is here [715](https://github.com/opensds/multi-cloud/issues/715)